test/widgets/profile_test.dart

```diff
@@ -96,9 +97,20 @@ void main() {
       deliveryEmail: 'testuser@example.com');
 
     await setupPage(tester, users: [user], pageUserId: user.userId);
+    await store.changeUserStatus(user.userId, UserStatusChange(
+      text: OptionSome('Busy'),
+      emoji: OptionSome(StatusEmoji(emojiName: 'working_on_it',
+        emojiCode: '1f6e0', reactionType: ReactionType.unicodeEmoji))));
+    await tester.pump();
 
     check(because: 'find user avatar', find.byType(Avatar).evaluate()).length.equals(1);
     check(because: 'find user name', find.text('test user').evaluate()).isNotEmpty();
+    final statusEmojiFinder = find.ancestor(of: find.text('\u{1f6e0}'),
+      matching: find.byType(UserStatusEmoji));
+    check(because: 'find user status emoji', statusEmojiFinder).findsOne();
+    check(tester.widget<UserStatusEmoji>(statusEmojiFinder)
+      .neverAnimate).isFalse();
+    check(because: 'find user status text', find.text('Busy')).findsOne();
```

Comment:
Let's put these checks into their own new test case, and leave this one as it is.

That will make both sets of test steps easier to understand than when they're combined into one does-everything test scenario like this.

That will also make this test a bit stronger: the existing test case will effectively check that if there is no user status, that doesn't cause this page to crash (e.g. because of a misplaced `!`, as in `userStatus.text!`).